learntools/core/asserts.py

```diff
@@ -1,31 +1,107 @@
+"""
+Some assertion helpers used in Problem.check implementations.
+
+TODO: standardize on some conventions for how these helpers are called, and how they
+present information to the user:
+    - When showing actual/expected values, do we use the result of str() or repr()?
+        - (Should we truncate the representation in case it's huge?)
+    - When showing types, str or repr?
+"""
+
 import os
+import numbers
+import math
+import functools
+import logging
 
 import pandas as pd
+import numpy as np
+
+def name_or_var(assert_fn):
```

Comment:
I don't understand this docstring, and possibly don't understand it well enough to articulate what I don't understand. 

It first, I'm confused that "A prose description" can be "the actual value being checked"?  

Is the purpose of this decorator to help fill in error messages?

A short usage example in the docstring may be helpful.

Reply:
> Assert helpers marked with this decorator take a "name" argument, which is a markdown string **representing** the actual value being checked.

This was actually a poor choice of words. I just pushed a change to replace "representing" with "describing". I also added an example to the docstring. But let me know if it's still not clear.

Reply:
The example there makes it really clear.  Thanks.